package.json

```
@@ -20,6 +20,7 @@
     "eslint": "eslint . --report-unused-disable-directives",
     "eslint-fix": "eslint --fix . --report-unused-disable-directives",
     "generate": "run-s generate:*",
+    "generate:distro-json-generate": "npx ts-node tools/distro-json-generate.ts",
```

```suggestion
    "generate:distro-json-generate": "ts-node tools/distro-json-generate.ts",
```
we've ts-node as dev dep. 

we probably should add the alternative typescript Compiler for ts-node to speedup the execution and convert all tools to TS again for better type safety.

so please do the ts conversation in a separate PR.